docs/metrics/accelerate.md

```diff
@@ -63,6 +63,15 @@ average(incident resolved timestamp - incident created timestamp)
 
 Pulse uses the maximum value over a period of time to display aggregate times to recover.
 
+<!--match-incident-deployment-start-->
+!!! note
+    We decided to avoid requiring a relationship between **incidents** and **deployments** or **changes** to simplify the data model reported.
```

Comment:
I find this sentence very confusing...
When I read it, I consider that incidents and deployments don't need to be related in any way. But, in fact, we require that they have the same "system", which can be seen as a relationship.

Reply:
Indeed. :+1: I re-analyzed all information and feedback received so far and made a completely new proposal for the notes.